pkg/utils/utils.go

```diff
@@ -0,0 +1,13 @@
+package utils
```

Comment:
what about avoiding a `utils` package and use `nodes` instead, similar to `labels`?

Reply:
Good idea :+1: